quinn-udp/src/unix.rs

```diff
@@ -49,12 +50,13 @@ impl UdpSocketState {
             last_send_error: Mutex::new(now.checked_sub(2 * IO_ERROR_LOG_INTERVAL).unwrap_or(now)),
             max_gso_segments: AtomicUsize::new(gso::max_gso_segments()),
             gro_segments: gro::gro_segments(),
+            ip_dontfrag_failing: false,
             sendmsg_einval: AtomicBool::new(false),
         }
     }
 
-    pub fn configure(sock: UdpSockRef<'_>) -> io::Result<()> {
-        init(sock.0)
+    pub fn configure(&mut self, sock: UdpSockRef<'_>) -> io::Result<()> {
+        init(sock.0, &mut self.ip_dontfrag_failing)
```

Comment:
I think it would be cleaner to just run an `io::Result<bool>` from `init()` instead of passing a `&mut bool` into it.

Reply:
Thanks for your commit, I have rebased the code.